### PR TITLE
 [MIG] hr_employee_lastnames: Migration to 14.0

### PR DESCRIPTION
**Pull request by luistorresm**

### CODE REVIEW

**Comment by luistorresm** [comment]
Hello @moylop260 

Could you review and merge this PR if you agree, please?

Regards

File: hr_employee_lastnames/README.rst
Comment (moylop260) [inline hr_employee_lastnames/README.rst]:
```suggestion
    :target: https://runbot.odoo-community.org/runbot/116/14.0
```

File: hr_employee_lastnames/README.rst
Comment (moylop260) [inline hr_employee_lastnames/README.rst]:
```suggestion
    :target: https://translation.odoo-community.org/projects/hr-14-0/hr-14-0-hr_employee_lastnames
```

File: hr_employee_lastnames/README.rst
Comment (moylop260) [inline hr_employee_lastnames/README.rst]:
```suggestion
This module is part of the `OCA/hr <https://github.com/OCA/hr/tree/14.0/hr_employee_lastnames>`_ project on GitHub.
```

**Review by moylop260 — CHANGES_REQUESTED** [review]
Please, check the `s/13.0/14.0/g` comments, please

File: hr_employee_lastnames/README.rst
Comment (emagdalenaC2i) [reply hr_employee_lastnames/README.rst]: @moylop260 the README file is autogenerated after merge

Comment (luistorresm) [reply hr_employee_lastnames/README.rst]: Thanks @emagdalenaC2i

@moylop260 

Was applied.

Regards

**Pull request merged**
